### DISCUSSION TITLE
How do i sort my posts on posts page? I'm using blogs theme

### DISCUSSION THREAD

**Discussion by shamimfahad** [shuding/nextra#2580] [Q&A] [answered]
I have a posts directory and all my posts are written in different md files that has the `type: post`, but they appear randomly/certain pattern on the posts page? but I want to define in which order they should appear in the posts page. how can I do that?

**Comment by jellli** [comment, marked as answer]
the posts are sorted by date by default.
you can try to add `date` field in your md frontmatter
